### PR TITLE
{vis}[GCCcore/6.4.0] cairo v1.14.12, GLib v2.54.3 + deps

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/deda0b1c7281e9b2bdfa2ce80026038e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2105.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d5a766e9a21d65b983b02ff7caf29b2d for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 15 out of 23 (4 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/1c8266291750fd848c612c6613a4cedb for a full test report.

**Comment by verdurin** [comment]
@boegel first error here is checksum failure for `XZ/5.2.3-GCCcore-6.4.0`. Could this be that old bug?

**Comment by boegel** [comment]
@verdurin I'm still getting `fd9ca16de1052aac899ad3495ad20dfa906c27b4a5070102a2ec35ca3a4740c1` as checksum for `xz-5.2.3.tar.bz2`, even after a re-download.

Maybe this is a genuine corrupted download on your end?

Can you check `/home/centos/.local/easybuild/sources/x/XZ/xz-5.2.3.tar.bz2`?

**Comment by verdurin** [comment]
@boegel looks like it was a genuine bad download. Build proceeding now.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (4 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/471a94d1c61c4888c0e7c7e3c9e1bbb1 for a full test report.

File: easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ('libxml2', libxml2_ver),
+    ('ncurses', '6.0'),
+]
+
+# use same binutils version that was used when building GCCcore toolchain
+builddependencies = [('binutils', '2.28')]
+
+configopts = '--without-emacs --with-libxml2-prefix=$EBROOTLIBXML2'
+
+moduleclass = 'vis'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb]: @boegel very minor nitpick - why is the `moduleclass` `vis`?

Comment (boegel) [reply easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb]: That's a very good question, it's clearly wrong...

I suggest we fix this in a followup PR (and also add a custom `sanity_check_paths` to the `gettext` easyconfigs, while we're at it)?

Comment (verdurin) [reply easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb]: Yes, agreed. I'll merge this one for now.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**